README.md

```diff
@@ -1,4 +1,4 @@
-# Mind Expanding Books
```

Comment:
Do you what is this change?

Reply:
Hello, what changes do I have to make?

Reply:
please accept my pr, i want to make progress in hacktoberfest

Reply:
@YashAmin-01 I mean, do you know why this line has changed? Can you undo the change?

Reply:
made the correction that you mentioned.